src/tinode.js

```diff
@@ -4716,6 +4730,10 @@ Topic.prototype = {
     // as placeholers for deleted ranges.
     // The messages are iterated by seq ID in ascending order.
     this._messages.forEach((data) => {
+      // Message not sent.
```

Comment:
Maybe expand the comment '// Do not create a gap between the last sent message and the first unsent.'

Reply:
Done.